Makefile

```diff
@@ -94,9 +94,14 @@ add-headers:
 # Corrects spelling errors
 .PHONY: misspell
 misspell:
-	go get github.com/client9/misspell
+	go get -u github.com/client9/misspell/cmd/misspell
 	# Ignore Kibana files (.json)
-	$(FIND) -not -path "*.json" -name '*' -exec misspell -w {} \;
+	$(FIND) \
+		-not -path "*.json" \
+		-not -path "*.log" \
+		-not -path "CHANGELOG*" \
```

Comment:
Does changelog have something inside which is "broken"?

Reply:
`./CHANGELOG.asciidoc:86:26: corrected "occured" to "occurred"`

But, actually, that shouldn't be ignored, my spelling pr fixes a bunch of things in that file (that was just added after my fixes were started...)

I'll remove this.